Fabric.Authorization.AccessControl/src/app/modules/access-control/grain-list/grain-list.component.ts

```diff
@@ -33,16 +33,18 @@ export class GrainListComponent implements OnInit {
 
   constructor(
     private route: ActivatedRoute,
-    private grainService: FabricAuthGrainService
+    private grainService: FabricAuthGrainService,
+    private router: Router
   ) {
     this.isGrainVisible = grainService.isGrainVisible();
     this.treeFlattener = new MatTreeFlattener(this.transformer, this._getLevel, this._isExpandable, this._getChildren);
     this.treeControl = new FlatTreeControl<GrainFlatNode>(this._getLevel, this._isExpandable);
     this.dataSource = new MatTreeFlatDataSource(this.treeControl, this.treeFlattener);
 
     const routeGrain = this.route.snapshot.params['grain'];
+    const routeSecurableItem = this.route.snapshot.params['securableItem'];
     if (!!routeGrain) {
-      this.setSelectedNode(routeGrain, this.route.snapshot.paramMap.get('securableItem'));
+      this.setSelectedNode(routeGrain, routeSecurableItem);
```

Comment:
Thanks for fixing this. I introduced the variable to replace that and didn't do it.